test/executor.test.js

```diff
@@ -271,6 +271,38 @@ describe('executor', function() {
         done();
       });
     });
+
+    it('should define `mixins/*` files', function() {
+      var modelBuilder = app.loopback.modelBuilder;
+      var registry = modelBuilder.mixins.mixins;
+      expect(registry).to.have.property('Example');
+      expect(registry).to.have.property('Foo');
+      expect(registry).to.have.property('TimeStamps');
+
+      var meta = modelBuilder.mixins.meta;
+      expect(meta).to.have.property('Example');
+      expect(meta).to.have.property('Foo');
+      expect(meta).to.have.property('TimeStamps');
+
+      expect(meta.Foo.name).to.equal('Foo');
+      expect(meta.Foo).to.have.property('filepath');
+
+      var description = 'Track create and update';
+
+      var options = {
+        postfix: {
+          type: 'string',
+          description: 'The postfix string',
+          default: 'At'
+        }
+      };
+
+      expect(meta.TimeStamps.name).to.equal('Timestamping');
+      expect(meta.TimeStamps).to.have.property('filepath');
+      expect(meta.TimeStamps.description).to.equal(description);
+      expect(meta.TimeStamps.options).to.eql(options);
+    });
```

Comment:
It would be nicer to create the test app from the test, rather than relying on a static fixture in `fixtures/simple-app`. However, in the case of the executor, I can live with simple-app too.

The test is testing too many things, please split it into multiple smaller and focused tests. A minimum is two tests, one testing the registry & meta keys, the other testing meta.TimeStamps.

I don't see how `meta.Foo` is different from `meta.TimeStamps` and how `meta.Foo` checks can fail when `meta.TimeStamps` checks pass. Could you please explain? If the fact that `meta.TimeStamps` checks are passing implies that `meta.Foo` checks pass too, them please remove `meta.Foo` checks as they are redundant.

Also add a comment explaining where are `description` and `options` coming from, or read them programatically in the test.

```js
var meta = require('./fixtures/simple-app/mixins/time-stamps.json');
expect(meta.TimeStamps.description).to.equal(meta.description);
// etc.
```